src/shared/components/palette/fixed-list/write-fixed-list-field.component.ts

```diff
@@ -11,19 +11,12 @@ export class WriteFixedListFieldComponent extends AbstractFieldWriteComponent im
   fixedListFormControl: FormControl;
 
   ngOnInit() {
-
-    /**
-     *
-     * Reassigning list_items from formatted_list when list_items is empty for DynamicList's
-     */
-    if (!this.caseField.list_items && this.caseField.formatted_value && this.caseField.formatted_value.list_items) {
-      this.caseField.list_items = this.caseField.formatted_value.list_items;
-    }
-
```

Comment:
This has been removed as the dynamic list now is in its own component so does not need this clause in here